File: ProblemSets1/ejer1.java
import java.util.Scanner;

public class ejer1 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, h, row = 0;
        n = sc.nextInt();
        h = sc.nextInt();
        int p[] = new int[n];
        for (int i = 0; i < n; i++) {
            p[i] = sc.nextInt();
        }
        for (int i = 0; i < n; i++) {
            if (p[i] <= h) row++;
            else if (p[i] > h) row = row + 2;
        }

        System.out.print(row);
    }
}

File: ProblemSets1/ejer1.c
#include <stdio.h>

int main() {

  int n, h;
  scanf("%d", &n);
  scanf("%d", &h);

  int p[n];
  int row=0;
  for (int i = 0; i < n; i++) {
    scanf("%d", &p[i]);
  }

  for (int i = 0; i < n; i++) {
    if (p[i] <= h) {
      row++;
    } else if (p[i] > h) {
      row = row + 2;
    }
  }
  printf("%d", row);
}
